base/tks/src/main/java/org/dogtagpki/server/tks/rest/v1/TPSConnectorService.java

```diff
@@ -302,22 +303,22 @@ public Response createSharedSecret(String id) {
                 throw new BadRequestException("Shared secret already exists");
             }
 
-            CryptoUtil.createSharedSecret(nickname,KeyGenAlgorithm.AES,AES_SESS_KEYSIZE);
+            CryptoUtil.createSharedSecret(nickname,KeyGenAlgorithm.AES,AES_SESS_KEYSIZE_256);
 
             TPSConnectorConfig tpsConfig = cs.getTPSConnectorConfig(id);
             tpsConfig.setNickname(nickname);
             cs.commit(true);
 
             //Create aes session sym key to wrap the shared secret.
-            SymmetricKey tempKey = CryptoUtil.createAESSessionKeyOnInternal(AES_SESS_KEYSIZE);
+            SymmetricKey tempKey = CryptoUtil.createAESSessionKeyOnInternal(AES_SESS_KEYSIZE_256);
 
             if (tempKey == null) {
                 return createNoContentResponse();
             }
 
             logger.debug("TPSConnectorService.createSharedSecret. about to export shared secret : " + nickname + " certs.length " + certs.length);
             logger.debug("TPSConnectorService.createSharedSecert cert: " + certs[certs.length -1]);
-            List<byte[]> listWrappedKeys = CryptoUtil.exportSharedSecret(nickname, certs[certs.length -1], tempKey, getUseOAEPKeyWrap());
+            List<byte[]> listWrappedKeys = CryptoUtil.exportSharedSecretWithAES(nickname, certs[certs.length -1], tempKey,getUseOAEPKeyWrap());
             logger.debug("TPSConnectorService.createSharedSecret. done exporting shared secret : " + nickname);
```

Comment:
I think the same fix should be applied to [TPSConnectorProcessor.java](https://github.com/dogtagpki/pki/blob/7407f408fd5329eab5620ed86cb889fce9a4c081/base/tks/src/main/java/org/dogtagpki/server/tks/rest/base/TPSConnectorProcessor.java#L268-L327) because in the future we should move to rest v2.

Reply:
OK, thanks!

Reply:
Concern has been addressed in recent push.